pkg/prometheus/common/config/http_config_test.go

```diff
@@ -824,6 +968,29 @@ func TestTLSRoundTripperRaces(t *testing.T) {
 	}
 }
 
+func TestHideHTTPClientConfigSecrets(t *testing.T) {
```

Comment:
Can you remove TestHideHTTPClientConfigSecrets for now?

Reply:
Yes, can you explain why? It works, why bother removing it?

Reply:
Forget above comment :D